### PR TITLE
New grab animations

### PR DESCRIPTION
**Pull request by skaljac**
Related issues:
https://github.com/webaverse/app/issues/1981
https://github.com/webaverse/app/issues/1873
https://github.com/webaverse/app/issues/1825

In this PR we have new animations for item grab action.
Animations are: 
grab forward
grab down
grab up
grab left
grab right

Also "group objects cannot be found when worn" is already solved by Avaer but we just need one simple null check and lantern/fairy can be picked up / droped without any problem.

Steps for testing:
Prepare scene with pickable objects at different heights.
Check object grab / pick up in directly in front of player, on players left side, on players right side.
Also place pickable object above players head, and on the floor. 
Appropriate animation should be shown.

Plase take a look at video below for reference

### CODE REVIEW

**Comment by skaljac** [comment]
https://user-images.githubusercontent.com/4487857/147165723-c9874aee-9904-4444-88c0-695ae258daf4.mp4

File: game.js [hunk 1]
```diff
@@ -705,6 +705,61 @@ const _gameUpdate = (timestamp, timeDiff) => {
...
+    if (distance < currentDistance) {
+      currentDistance = distance;
+      currentAnimation = "grab_right";
+    }
+
+    if (localPlayer.getAction('activate')) {
+      localPlayer.getAction('activate').animationName = currentAnimation;
+    }
+
+    return {distance:currentDistance,
```
Comment (avaer) [inline game.js:759]: There is no need to return here; it counts as an allocation and isn't used.

Comment (skaljac) [reply game.js]: I have removed this return statement now

**Comment by amnaarshed4224** [comment]
Issues:
1-The animation of grabing itself seems right but it seems to have some issues with direction of grab. (the direction in which hand moves to pick an object)
2-When avatar is facing opposite the object the direction of grab is totally opposite.
3-When tried to pick object diagonally same issue of direction persists
4-Trying to grab the item above or in any direction(below, left right), If the avatar is standing at bit distance, the hand doesn't seems to be going in the direction of the object.

https://user-images.githubusercontent.com/35926530/147277095-4f91888f-8856-4069-94c9-375934b4b851.mp4


https://user-images.githubusercontent.com/35926530/147277150-e7d5a661-d579-4473-bf2a-deb3bcab7c70.mp4

**Comment by avaer** [comment]
Yeah I tested this and I think the grab directions might not account for object rotation.

**Comment by skaljac** [comment]
I have change the way how this works. If the player is not facing item, he will not get E button.

I hope this is what ok now. If not please let me know what should I change.

https://user-images.githubusercontent.com/4487857/147289027-88c1296a-e0c0-4077-93ae-a45d029a1ecb.mp4

**Comment by avaer** [comment]
Tried it and it works fine; it certainly looks a lot better than `master`.

We can continue improvements there.

**Pull request merged**